packages/fhir-router/src/graphql/graphql.test.ts

```diff
@@ -552,7 +552,7 @@ describe('GraphQL', () => {
 
   test('Max depth', async () => {
     // The definition of "depth" is a little abstract in GraphQL
-    // We use "selection", which, in a well formatted query, is the level of indentation
+    // We use field depth, where fragment expansion does not contribute to depth
```

Comment:
`does not contribute` / `does contribute`?

Reply:
Ah, I see how this is ambiguous: the expansion itself (which does have a selection set) doesn't count, but the fields inside the expansion do count